src/components/ScheduleFieldset.vue

```diff
@@ -4,6 +4,7 @@
       {{ internalValue.toString({ verbose: true }) }}
     </div>
 
+    <!-- todo: audit [cchoy] -->
```

Comment:
not quite sure what to do with this one 🤔

Reply:
I'm not sure where this is used (on my phone or I'd look) but I feel like this component should just accept a deployment rather than a schedule. I'll have to look at why it's accepting just a schedule